cmd/generate.go

```diff
@@ -163,6 +156,7 @@ function start_cockroach() {
 # Execute setup.sh script on the cluster to configure it
 function setup_cluster() {
 	roachprod run "$1" sudo ./scripts/gen/setup.sh "$CLOUD"
+	roachprod run "$1":1 -- cpufetch -s legacy|awk -F"@" '{print $NF}'|tr -d ' '|awk NF > "$logdir"/"$1"_cpu_info.txt
```

Comment:
Since `cpufetch` doesn't (always?) report max. frequency, should we also append this?

```
lscpu |grep "MHz"
CPU MHz:                         2299.998
```

Reply:
Yep added it.